src/main/java/seedu/address/logic/commands/AddPartToServiceCommand.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PART_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_QUANTITY;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_SERVICE_ID;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Manages adding of parts to services
+ */
+public class AddPartToServiceCommand extends Command {
+    public static final String COMMAND_WORD = "addpartservice";
```

Comment:
Minor suggestion: maybe `addservicepart` or `addparttoservice` is better?

Reply:
I'll change to `addservicepart`